.pre-commit-config.yaml

```diff
@@ -41,6 +41,16 @@ repos:
     hooks:
       - id: ruff
         args: [ --fix, --exit-non-zero-on-fix ]
+      - id: ruff-format
+        files: |
+            (?x)^(
+              bazel/|
+              cpp/|
+              doc/|
+              docker/|
+              java/|
```

Comment:
All these are included as well?

Reply:
Yes. Because the output of `black` and `ruff-format` is already the same for those folders.